client/src/main/java/io/pravega/client/stream/impl/ControllerImpl.java

```diff
@@ -968,6 +968,7 @@ public void onNext(T value) {
         @Override
         public void onError(Throwable t) {
             log.warn("gRPC call failed with server error: {}", t.getMessage());
```

Comment:
It would be better if this told us more about which message got the failure. Maybe we could include the request in the log here.

Reply:
Fixed. Added more details to the log.